DNN Platform/Library/Entities/Host/MailSettings.cs

```diff
@@ -13,7 +13,7 @@ namespace DotNetNuke.Entities.Host;
 /// <summary>The default <see cref="IMailSettings"/> implementation.</summary>
 /// <param name="hostSettingsService">The host settings service.</param>
 /// <param name="hostSettings">The host settings.</param>
-public class MailSettings(IHostSettingsService hostSettingsService, IHostSettings hostSettings) : IMailSettings
+public class MailSettings(IHostSettingsService hostSettingsService, IHostSettings hostSettings, IPortalController portalController) : IMailSettings

```

Comment:
Do we need to add an overload with the original parameters since this class is public ?

Reply:
This class was added in https://github.com/dnnsoftware/Dnn.Platform/pull/6538, which also targeted 10.0.2, so there shouldn't be a need for an additional overload.

Reply:
Perfect, thanks.